build/docker/intel-idxd-config-initcontainer.Dockerfile

```diff
@@ -15,6 +15,8 @@
 ## limitations under the License.
 ###
 FROM debian:unstable-slim
+COPY ./LICENSE .
+RUN install -D /LICENSE /licenses/intel-device-plugins-for-kubernetes/LICENSE && rm /LICENSE
```

Comment:
`COPY` is enough: "If <dest> doesn't exist, it's created, along with all missing directories in its path."

Reply:
OK ;) Thanks!